core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -245,32 +245,31 @@ var TopicsAndSkillsDashboardPage = function() {
   this.filterSkillsByStatus = async function(status) {
     await action.click(
       'Skill Dashboard status filter', skillStatusFilterDropdown);
-    await (
-      await browser.driver.switchTo().activeElement()
-    ).sendKeys(status + '\n');
+    var dropdownOption = element(
+      by.cssContainingText('mat-option .mat-option-text', status));
+    await action.click('Skill status filter option', dropdownOption);
```

Comment:
In the element name (first argument), can you include the value of `status`?

Reply:
done